docs/integrations/databricks.md

```diff
@@ -28,7 +28,8 @@ For Databricks to work with lakeFS, set the S3 Hadoop configuration to the lakeF
 ```
 spark.hadoop.fs.s3a.bucket.<repo-name>.access.key AKIAIOSFODNN7EXAMPLE
 spark.hadoop.fs.s3a.bucket.<repo-name>.secret.key wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
-spark.hadoop.fs.s3a.bucket.<repo-name>.endpoint https://s3.lakefs.example.com
+spark.hadoop.fs.s3a.bucket.<repo-name>.endpoint https://lakefs.example.com
+spark.hadoop.fs.s3a.path.style.access true
```

Comment:
this one doesn't have a bucket-level setting?

Reply:
it might but if you're requiring this behavior from the SDK might as well make it consistent?

Reply:
thought the above example is how to make it specific - but I'll agree as long it is the preferred way anyway.